services/src/pro/contexts/migrations/migration_0000_initial.rs

```diff
@@ -0,0 +1,122 @@
+use async_trait::async_trait;
+use pwhash::bcrypt;
+use tokio_postgres::Transaction;
+
+use super::database_migration::{ProMigration, ProMigrationImpl};
+use crate::{
+    contexts::Migration0000Initial,
+    error::Result,
+    layers::{
+        add_from_directory::UNSORTED_COLLECTION_ID, storage::INTERNAL_LAYER_DB_ROOT_COLLECTION_ID,
+    },
+    pro::permissions::Role,
+    util::config::get_config_element,
+};
+
+#[async_trait]
+impl ProMigration for ProMigrationImpl<Migration0000Initial> {
+    async fn pro_migrate(&self, tx: &Transaction<'_>) -> Result<()> {
+        let user_config = get_config_element::<crate::pro::util::config::User>()?;
+
+        tx.batch_execute(include_str!("migration_0000_initial.sql"))
+            .await?;
+
+        let stmt = tx
+            .prepare(
+                r#"
+            INSERT INTO roles (id, name) VALUES
+                ($1, 'admin'),
+                ($2, 'user'),
+                ($3, 'anonymous');"#,
+            )
+            .await?;
+
+        tx.execute(
+            &stmt,
+            &[
+                &Role::admin_role_id(),
+                &Role::registered_user_role_id(),
+                &Role::anonymous_role_id(),
```

Comment:
Here, we also have the problem that this values can change and then we have a different state 0.

Reply:
so should I add a new constant here and use that instead?

Reply:
Yeah, it must be locally, I guess

Reply:
045393894029fd54081e75acd1ba1c940f6bd3b8